src/sass/component/_research-banner.scss

```diff
@@ -0,0 +1,35 @@
+.research-banner {
+    @include AU-space(padding, 2unit);
+    @include AU-space(margin-bottom, 1unit);
+    background: #ebebeb66;
+
+    .research-banner__image {
+        display: none;
+        @include AU-space(margin-top, 2unit);
+        content: url("http://placehold.it/700x200")
```

Comment:
We don't make friends with content injecting stuff this is bad 👍

Reply:
fixed